Ladybird/CMakeLists.txt

```diff
@@ -34,6 +34,17 @@ function(create_ladybird_bundle target_name)
         add_custom_command(TARGET ${target_name} POST_BUILD
             COMMAND "${CMAKE_COMMAND}" -E create_symlink "${CMAKE_LIBRARY_OUTPUT_DIRECTORY}" "${bundle_dir}/Contents/lib"
         )
+
+        if (CMAKE_BUILD_TYPE STREQUAL "Debug")
```

Comment:
Would it be better to have this as a 'if not RELEASE, apply the entitlement'?

It's definitely not a good idea to "ship" with this guy on.

Reply:
hmm that might be better. Depends on how you want RelWithDebInfo to behave?

Reply:
eh. I'll just disable it for Release and RelWithDebInfo